### ISSUE TITLE
Midi Audio Support

### ISSUE THREAD

**Issue by eliudfromkenya** [jfversluis/Plugin.Maui.Audio#125]
Can it be enhanced to support midi music,  one that can receive and play midi event real time?
.NET, more so MAUI really requires one.

**Comment by jfversluis** [comment]
I don't know much about MIDI to be honest. I would think that is out of scope for this plugin or at least, I don't think I will be adding it anytime soon. But if you have ideas or want to contribute, feel free to start discussing how.

**Comment by eliudfromkenya** [comment]
It's ok, I will check through and to see if there's something I can do then I will get back to you.

But any person with an idea how to do it please assist.

**Comment by jfversluis** [comment]
I don't think this will happen anytime soon as mentioned. So closing this for now. If anyone finds this and knows MIDI, feel free to contact us.
